src/components/RouterComponent.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+
+import QuestionList from './QuestionList';
+import QuestionLists from './QuestionLists';
+
+export default function RouterComponent(): JSX.Element {
```

Comment:
Check docs/FOLDER_STRUCTURE.MD. You should move router to to own file in root of src folder